submissions/crealime/frogger-game/js/app.js

```diff
@@ -0,0 +1,273 @@
+const X_SIZE = 101
+const Y_SIZE = 83
+const ROWS = 6
+const COLUMNS = 5
+const START_ROW = 5
+const START_COLUMN = 2
+const WIDTH_FIELD = COLUMNS * X_SIZE
+const HEIGHT_FIELD = ROWS * Y_SIZE
+const Y_OFFSET_PLAYER = 35
+const Y_OFFSET_ENEMY = 35
+const X_OFFSET_ENEMY = 20
+const START_PLAYER_X_POSITION = X_SIZE * START_COLUMN
+const START_PLAYER_Y_POSITION = Y_SIZE * START_ROW - Y_OFFSET_PLAYER
```

Comment:
You may want to group constants per their common features.
e.g.
```
const FIELD = {
  rows: 5,
  cols: 6,
}

const FIELD_PIXEL_BASED_PROPS = {
  width: 530,
  height: 999,
}

const FIELD = {
  PIXEL_BASED: {
   TILE: {
      width: 50,
      height: 110,
   }
  },
  ROW: {
    count: 7,
    start: 5,
  },
  COL: {
    count: 8,
    start: WHATEVER * 5,
  },
}

const FIELD.PIXEL_BASED.DIMENSIONS = {
  width: FIELD.PIXEL_BASED.TILE.width * FIELD.COL.count,
  height: // .similar calcs
};


```
Look, how reads full path `FIELD.PIXEL_BASED.width`. Practically, you replace some underscores with dots, yet the constants definition has some logical structure.
The above is a quick example (so naming is not well elaborated) to give you some ideas and illustrate how we can leverage nested data to put data into some logical structure yet keeping things readable.

Also it might be more reasonable to have essential numbers in a constant object different from a constant object that contains calculated constants.